engine/apps/api/views/schedule.py

```diff
@@ -165,6 +167,8 @@ def get_queryset(self):
             queryset = self.serializer_class.setup_eager_loading(queryset)
         if filter_by_type is not None and filter_by_type in SCHEDULE_TYPE_TO_CLASS:
             queryset = queryset.filter().instance_of(SCHEDULE_TYPE_TO_CLASS[filter_by_type])
+        if used is not None:
+            queryset = queryset.filter(escalation_policies__isnull=not used).distinct()
```

Comment:
Can we check num_escalation_chains annotation instead of escalation_policies, I think it should be faster

Reply:
Thanks, I'll check this out!
In this particular case there are no annotations when `is_short_request` is set to `True` though. I'm not sure if `is_short_request` is used at all, I'll check this & update in a separate PR if it's not used.